### PR TITLE
Add freeform governance proposals to the governance spec

### LINKED ISSUE #700: Update spec for freeform governance proposals
In order to start development on the 💦 feature of 'freeform governance proposals' we need the spec merging to include information for the team to review and start development:

- [ ] Spec updated to detail the feature for development
- [ ] Acceptance criteria so that the feature can be tested (@vega-paul)
- [ ] Detail any QA scenarios for feature files (if applicable @cpuja84)

### PR DESCRIPTION
**Pull request by davidsiska-vega**

### CODE REVIEW

File: protocol/0028-governance.md [hunk 1]
```diff
@@ -266,6 +266,22 @@ transfer_amount == min(
     proposal.amount )
 ```
 
+## 6. Freeform governance proposal
```
Comment (edd) [inline protocol/0028-governance.md:269]: **Question (blocking)** Is there any requirement for the network to verify the hash of the text file?

My assumption is no, but this should be explicit.

**Review by edd — CHANGES_REQUESTED** [review]
One question that needs an answer, otherwise this looks fine.

File: protocol/0028-governance.md [hunk 2]
```diff
@@ -266,6 +266,22 @@ transfer_amount == min(
...
+
```
Comment (edd) [inline protocol/0028-governance.md:270]: Can we specify a max length for the proposal text fields?

Comment (davidsiska-vega) [reply protocol/0028-governance.md:270]: Made it 255 characters, feel free to change.

File: protocol/0028-governance.md (hunk 1 above)
Comment (davidsiska-vega) [reply protocol/0028-governance.md:269]: You are correct and I've made the text explicit.

**Review by edd — APPROVED** [review]

**Pull request merged**